csrc/scheduler/tools/inlining.cpp

```diff
@@ -193,6 +195,21 @@ size_t MaxPosCalculator::getMaxProducerPosFromConsumer(
     }
     return producer->nDims();
   } else {
+    std::unordered_set<ValGroup> loop_path_groups;
+    if (consumer->definition()->isA<MmaOp>()) {
+      // Get ValGroups between producer and consumer loop in the inlining graph
```

Comment:
Maybe it would make more sense to start by using a `PairwiseLogicalDomainMap` to find mapped ID groups between producer and consumer, then doing BFS to find the path from this to both loop domain val groups. That way we could avoid missing loop groups in both the producer _and_ consumer. That case won't occur for `MmaOp` but it might be clearer than traversing from producer loop to consumer loop since it's not immediately clear why we go that direction.